storage/innobase/xtrabackup/src/redo_log.cc

```diff
@@ -132,6 +132,7 @@ bool Redo_Log_Reader::is_error() const { return (m_error); }
 
 /** scan redo log files form server directory and update log.m_files.
 @param[in,out]  desired_lsn             LSN that triggered file reopening.
+if LSN = 0 PXB ony care about opening a redo file
```

Comment:
if LSN == 0, PXB opens the redo log file and checks if the redo is disabled or not.

Reply:
fixed